beanstalk_worker/worker_start_build.py

```diff
@@ -116,10 +118,6 @@ def start_build(job_details):
         job = bs.reserve()
         job_details = json.loads(job.body)
         result = start_build(job_details)
-        if result == 0:
-            debug_log("Build is successful deleting the job")
-            job.delete()
-        else:
-            debug_log("Job was not succesfull and returned to tube")
```

Comment:
We do not need this now as we are deciding success and failed while getting the status only. And if we keep this worker goes in infinite loop for the failed ones